submissions/OpemipoBanso/CSS/HTML and CSS tasks/html/kasuwa homepage.html

```diff
@@ -0,0 +1,526 @@
+                            <p>
+                                Sheep <br> from N8999
+                            </p>
+                        </a>
+                                    
+                    </div>
+                    <div class="tsitem">
+                                    
+                        <a href="">
+                            <img src="C:\Users\USER\Documents\Kasuwa_files\css\images\lst-5.jpeg" alt="Avocados">
+                            <p>
+                                Goats <br> from 9999
+                            </p>
+                        </a>
+                    </div>
+                                    
+            
+            
+                </div>
+                
+            </div>
+
+            <div class="top-selling-container">
+                <div class="top-selling-items">
+                    <h2>
+                        Your recent views
+                    </h2>
+                    <div class="tsitem">
+                                        
+                        <a href="">
+                            <img src="C:\Users\USER\Documents\Kasuwa_files\css\images\rec-1.jpg" alt="Avocados">
+                            <p>
+                                Sweet Corn <br> from N199
+                            </p>
+                        </a>
+                                        
+                
+                    </div> 
+                    <div class="tsitem">
+                                        
+                        <a href="">
+                            <img src="C:\Users\USER\Documents\Kasuwa_files\css\images\rec-6.jpg" alt="Avocados">
+                            <p>
+                                Fish <br> from N499
+                            </p>
+                        </a>
+                    </div>
+                    <div class="tsitem">
+                                        
+                        <a href="">
+                            <img src="C:\Users\USER\Documents\Kasuwa_files\css\images\rec-3.jpg" alt="Avocados">
+                            <p>
+                                Wheat <br> from N899
+                            </p>
+                        </a>
+                                    
+                    </div>
+                    <div class="tsitem">
+                                        
+                        <a href="">
+                            <img src="C:\Users\USER\Documents\Kasuwa_files\css\images\rec-4.jpeg" alt="Avocados">
+                            <p>
+                                Carrots <br> from N499
+                            </p>
+                        </a>
+                                        
+                    </div>
+                    <div class="tsitem">
+                                        
+                        <a href="">
+                            <img src="C:\Users\USER\Documents\Kasuwa_files\css\images\rec-2.jpg" alt="Avocados">
```

Comment:
This reference is too absolute. This won't work on another user's system. @Verdantcoderr . Find a way to make the src value "css/images/image_file" or images/image_file".

Reply:
Please do this for other img src